web/src/components/DateField.js

```diff
@@ -29,19 +29,33 @@ const joinDate = ({ day, month, year }) => {
 };
 
 const DateField = ({ value, onChange, ...rest }) => {
+  const [errorInfo, setErrorInfo] = useState({
+    errorMessage: [],
+    dayInvalid: false,
+    monthInvalid: false,
+    yearInvalid: false
+  });
+
   const dateParts = splitDate(value);
 
-  const errorInfo = useMemo(() => {
+  const getErrorMsg = () => {
```

Comment:
ESLint is complaining about this not returning; I'm not sure if I need to suppress the warning or rework this somehow.

Reply:
You can change the two `return null;` (lines 44 and 54) to just `return;` and that should fix it.  The problem is that the function *sometimes* returns a value (`null`), but sometimes doesn't return anything. It needs to either never return anything or always return something.